README.md

```diff
@@ -1,2 +1,41 @@
-# 1fst-capstone-project
-This is my first capstone project into Microverse's program
+# My First Capstone Project
+
+> This is the best way that I can show you my work at: https://dangelo-jan.github.io/1fst-capstone-project/
+
```

Comment:
[OPTIONAL]
- It would be better to add a descriptive description of your project to README to document your project professionally.